package.json

```diff
@@ -15,10 +15,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "^14.0.0",
+    "@angular/cdk": "^14.2.2",
     "@angular/common": "^14.0.0",
     "@angular/compiler": "^14.0.0",
     "@angular/core": "^14.0.0",
     "@angular/forms": "^14.0.0",
+    "@angular/material": "^14.2.2",
```

Comment:
Why we installed material ? we have bootstrap in other modules, can we keep bootstrap in all the modules for consistency ?

Also there is chance for some scenarios CSS/styling may get overlapped for material and bootstrap

Reply:
With the interest of time, I thought the material would be nice because the Bootstrap table needs a lot more CSS  changes to get done.